Smarty/templates/com_vtiger_workflow/taskforms/VTEntityMethodTask.tpl

```diff
@@ -8,17 +8,30 @@
  * All Rights Reserved.
  ************************************************************************************/
 -->*}
+
```

Comment:
in the future do not add empty lines like this

Reply:
Alright